modules/installation-guide/examples/system-variables.adoc

```diff
@@ -1,22 +1,22 @@
 pass:[<!-- vale off -->]
 
-[id="che-server"]
-= Che server
+[id="{prod-id-short}-server"]
+= {prod-short} server
 
-.Che server 
+.{prod-short} server 
 ,=== 
  Environment Variable Name,Default value, Description 
  
  `+CHE_DATABASE+`,"`+${che.home}/storage+`","Folder where {prod-short} stores internal data objects." 
  `+CHE_API+`,"`+http://${CHE_HOST}:${CHE_PORT}/api+`","API service. Browsers initiate REST communications to {prod-short} server with this URL." 
- `+CHE_API_INTERNAL+`,"`+http://${CHE_HOST}:${CHE_PORT}/api+`","API service internal network url. Back-end services should initiate REST communications to {prod-short} server with this URL" 
- `+CHE_WEBSOCKET_ENDPOINT+`,"`+ws://${CHE_HOST}:${CHE_PORT}/api/websocket+`","{prod-short} websocket major endpoint. Provides basic communication endpoint for major websocket interactions and messaging." 
+ `+CHE_API_INTERNAL+`,"`+http://${CHE_HOST}:${CHE_PORT}/api+`","API service internal network URL. Back-end services should initiate REST communications to {prod-short} server with this URL" 
+ `+CHE_WEBSOCKET_ENDPOINT+`,"`+ws://${CHE_HOST}:${CHE_PORT}/api/websocket+`","{prod-short} WebSocket major endpoint. Provides basic communication endpoint for major WebSocket interactions and messaging." 
  `+CHE_WORKSPACE_PROJECTS_STORAGE+`,"`+/projects+`","Your projects are synchronized from the {prod-short} server into the machine running each workspace. This is the directory in the machine where your projects are placed." 
- `+CHE_WORKSPACE_PROJECTS_STORAGE_DEFAULT_SIZE+`,"`+1Gi+`","Used when {orch-name}-type components in a devfile request project PVC creation (Applied in case of 'unique' and 'per workspace' PVC strategy. In case of the 'common' PVC strategy, it is rewritten with the value of the `che.infra.kubernetes.pvc.quantity` property.)" 
+ `+CHE_WORKSPACE_PROJECTS_STORAGE_DEFAULT_SIZE+`,"`+1Gi+`","Used when {orch-name}-type components in a devfile request project PVC creation (Applied in case of `unique` and `per workspace` PVC strategy. In case of the `common` PVC strategy, it is rewritten with the value of the `che.infra.kubernetes.pvc.quantity` property.)" 
  `+CHE_WORKSPACE_LOGS_ROOT__DIR+`,"`+/workspace_logs+`","Defines the directory inside the machine where all the workspace logs are placed. Provide this value into the machine, for example, as an environment variable. This is to ensure that agent developers can use this directory to back up agent logs." 
- `+CHE_WORKSPACE_HTTP__PROXY+`,"","Configures proxies used by runtimes powering workspaces." 
- `+CHE_WORKSPACE_HTTPS__PROXY+`,"","Configuresproxies used by runtimes powering workspaces." 
- `+CHE_WORKSPACE_NO__PROXY+`,"","Configuresproxiesused by runtimes powering workspaces." 
+ `+CHE_WORKSPACE_HTTP__PROXY+`,"","Configures environment variable HTTP_PROXY to a specified value in containers powering workspaces." 
```

Comment:
`Sets HTTP_PROXY environment variable to specified value in containers used for workspaces` ?